client/src/components/Footer.jsx

```diff
@@ -0,0 +1,50 @@
+import { NavLink } from 'react-router-dom';
+import { ReactComponent as Search } from '../icons/search.svg';
+import { ReactComponent as Add } from '../icons/add.svg';
```

Comment:
```
import { ReactComponent as Search } from '../icons/search.svg';
import { ReactComponent as Add } from '../icons/add.svg';
```

You don't use these for now or? 🤔 Then you should remove unnecessary imports as these could also hinder a successful vercel deployment.